tests/tinytest.R

```diff
@@ -43,5 +43,5 @@ if (requireNamespace("tinytest", quietly=TRUE)) {
     ## there are several more granular ways to test files in a tinytest directory,
     ## see its package vignette; tests can also run once the package is installed
     ## using the same command `test_package(pkgName)`, or by director or file
-    tinytest::test_package("Rcpp", ncpu=getOption("Ncpus", 1))
+    tinytest::test_package("Rcpp", ncpu=getOption("Ncpus"))
```

Comment:
I noticed that `tinytest` will use a parallel worker whenever Ncpus is set to a numeric value; leaving it NULL / unset lets us run tests (and see test output directly) in the current R session. Does changing this default seem reasonable?

Reply:
I was a fan of parallel runs (I may even have been the one who suggested it to Mark) but I no longer run them in many packages.  I guess I do / did here because I happen to have `getOption("Ncpus")` set (helps with parallel install.packages / update.packages too) .  But removing the trailing ", 1" is IMHO a bad idea as it removes the default and `getOption()`. If I try NULL explicitly I get rubbish:

```sh
edd@rob:~/git/rcpp(master)$ Rscript -e 'tinytest::test_package("Rcpp", NULL)'
Error in dir.exists(testdir) : invalid filename argument
Calls: <Anonymous> -> dir.exists
Execution halted
edd@rob:~/git/rcpp(master)$ 
```

My personal goto I use all the time is `rcc.r` from littler and it just calls `rcmdcheck::rcmdcheck()`.  So I would be fine with just simplifying this to 

```suggestion
    tinytest::test_package("Rcpp")
```